cmake/thirdparty/SetupMPI.cmake

```diff
@@ -33,6 +33,29 @@ set(_mpi_link_flags )
 
 message(STATUS "Enable FindMPI:  ${ENABLE_FIND_MPI}")
 
+#
+# TODO: should this be BLT_ENABLE_FIND_MPI?
+#
+if(ENABLE_FIND_MPI)
```

Comment:
Can you remove line 34 if you want to add this?

Yea this was a very early option.  It should be prefixed but I've been hesitant to change it due to legacy.  But honestly no one uses that I know anymore.  The main use case was BGQ.

Reply:
removed line 34 and the TODO